rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/update/85_fields_meta.yml

```diff
@@ -2,8 +2,7 @@
 "Metadata Fields":
 
   - skip:
-      version:  "all"
-      reason:   "Update doesn't return metadata fields, waiting for #3259"
+      awaits_fix: "Update doesn't return metadata fields, waiting for #3259"
```

Comment:
Nit: can we find the URL? (3259 also seems wrong)

Reply:
same as above, this is not the goal of the PR

Reply:
Yeah, I understand that, it could be so time consuming. Just make a note somewhere so there is a chance we don't forget it :)